iota-client/Cargo.toml

```diff
@@ -19,28 +19,34 @@ bee-pow = { git = "https://github.com/iotaledger/bee.git", rev = "e8cafb6591ec0c
 bee-common = { git = "https://github.com/iotaledger/bee.git", branch = "dev" }
 bee-crypto = { git = "https://github.com/iotaledger/bee.git", rev = "c42171ff33c80cc2efb183e244dc79b7f58d9ac4" }
 iota-crypto = { git = "https://github.com/iotaledger/crypto.rs.git", rev = "b849861b86c3f7357b7477de4253b7352b363627", features = ["std", "blake2b", "ed25519", "random", "slip10"]}
-ureq = { version = "2.0", features = ["json"], optional = true }
-reqwest = { version = "0.11", features = ["json", "rustls-tls", "blocking"], default-features = false, optional = true }
 url = "2.2"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 hex = "0.4"
-tokio = { version = "1.1", features = ["macros", "sync", "rt-multi-thread", "rt"] }
 thiserror = "1.0"
 num_cpus = "1.13"
-rusqlite = { version = "0.24", features = ["bundled"], optional = true }
-async-trait = "0.1"
 log= "0.4"
+regex = "1.4"
+
+# sync
+ureq = { version = "2.0", features = ["json"], optional = true }
+# async
+tokio = { version = "1.1", features = ["macros", "sync", "rt-multi-thread", "rt"] }
+reqwest = { version = "0.11", features = ["json", "rustls-tls", "blocking"], default-features = false, optional = true }
+futures = { version = "0.3", optional = true }
 
 # MQTT
 rumqttc = { version = "0.5", optional = true }
-regex = { version = "1.4", optional = true }
+# also used for storage
 once_cell = { version = "1.5", optional = true }
-futures = { version = "0.3", optional = true }
+
+#storage
+async-trait = "0.1"
+rusqlite = { version = "0.24", features = ["bundled"], optional = true }
 
 [features]
-default = ["async"]
+default = ["async", "futures"]
```

Comment:
is `futures` dependency here still used? couldn't find it

Reply:
It's used here https://github.com/iotaledger/iota.rs/blob/9d44e7ccc0b9dc8f8392e9d717f3af4a8d893846/iota-client/src/node_manager.rs#L133

Reply:
`futures` seems like a required dependency, not an optional one